backend/validators/social.validator.ts

```diff
@@ -0,0 +1,20 @@
+import Joi from 'joi';
+
+const schema = {
+  post: Joi.object({
+    name: Joi.string().required(),
+    image: Joi.string().required(),
+    active: Joi.boolean().required()
```

Comment:
Active should not be specified in new creation. It should be always active after creating